server/package.json

```diff
@@ -4,13 +4,8 @@
   "description": "100Devs Social Network",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js",
-    "dev": "concurrently \"npm:watch-client\" \"npm:watch-server\"",
-    "watch-server": "nodemon server.js",
-    "watch-client": "npm --prefix frontend start",
-    "postinstall": "npm --prefix frontend install && npm run build",
-    "build": "rm -rf build && cd ./frontend && npm run build --prod && cp -r build ../",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node server/server.js",
+    "dev": "nodemon server/server.js"
```

Comment:
I feel like `server/` is redudant, it will work with just `server.js`. Though it can be change when we overhaul to npm scripts again.

Reply:
Yeah, I forgot to remove that, since I was editing it on my phone.

Reply:
@JoeDravarol won't this remove our ability to build the front end? Also, I liked concurrently, but I'll download this later tonight or tomorrow to see how it functions.

Reply:
@Caleb-Cohen Unfortuantely yes, but I can look into fixing that.

Reply:
> @Caleb-Cohen Unfortuantely yes, but I can look into fixing that.

There's no right or wrong way of structuring the project but I think we should pull out all of the package.json file into the root folder, so that it'll be easier to execute the scripts at the root directory (How we currently have it setup) without the need to `cd` into multiple directories.

![image](https://user-images.githubusercontent.com/48932159/201496163-a9e605c9-9166-455d-8791-8ea749a85856.png)